### DISCUSSION TITLE
How to use Watchers?

### DISCUSSION THREAD

**Discussion by akadawa** [TheNickOfTime/handbrake-web#229] [Q&A] [answered]
Hi,

I actually thought I had to configure the watcher like this:

![image](https://github.com/user-attachments/assets/e16b7515-4495-4a18-ab01-7a9ff3f9de38)

it doesn't seem to work that way. so i have adapted the rule:
![image](https://github.com/user-attachments/assets/7de7685e-2101-4fcb-8380-4baf57ec2c43)

does not seem to be correct either.

how do i have to set the worker so that it automatically converts the videos? Does Queue Start Queue have to be active for the worker to be active?

**Comment by TheNickOfTime** [comment, marked as answer]
Hi, yes the queue does need to be started for workers to pick up jobs. Watchers simply automatically add jobs to the queue, so if your queue is not active, the jobs will not process. Additionally, as a point of clarification, watchers will not create jobs for existing files - only for files that are added. If you have existing files in your input directory, you can bulk add jobs via the create job dialogue in the queue section :)

If you have issues with watchers beyond this, please close this discussion and open a bug report [here](https://github.com/TheNickOfTime/handbrake-web/issues/new/choose).
